crates/fuel-core/src/graphql_api/storage/da_compression/v2.rs

```diff
@@ -0,0 +1,11 @@
+pub mod address;
+pub mod asset_id;
+pub mod contract_id;
+pub mod predicate_code;
+pub mod script_code;
+
+pub use address::*;
+pub use asset_id::*;
+pub use contract_id::*;
+pub use predicate_code::*;
+pub use script_code::*;
```

Comment:
I'd prefer to skip these wildcard re-exports and just use the full qualified path since the modules are public. I only see this used currently in docstring links in `storage.rs`.

Reply:
addressed in 6958329529